app/Console/Commands/GameIntervalCommand.php

```diff
@@ -46,7 +47,8 @@ public function handle()
         $gameController = new GameController();
         $lastUpdates = [];
 
-        while(1<2){
+        while (true) {
+            $game_ended = false;
```

Comment:
Ik zou deze variabel binnen de `foreach` declareren, dan weet je zeker over welke game je het hebt en voorkom je mogelijke bugs. Hiernaast is deze variable een beetje dubbel met `$gameTimeExpired`, deze is nu waarschijnlijk overbodig.